src/core/middleware.js

```diff
@@ -81,3 +81,15 @@ export function prefixMiddleWare(req, res, next, { _config = config } = {}) {
 
   return next();
 }
+
+export function trailingSlashesMiddleware(req, res, next) {
+  const URLParts = req.originalUrl.split('?');
```

Comment:
Good question. I tried it with multiple `?` in the URL (https://github.com/mozilla/addons-frontend/pull/1819/files#diff-6b5535b4fe81a5387879c0140e976318R184), but I'll see if `url.parse` will work fine (I'm sure they've tested their code more than I did!)